qubes/storage/__init__.py

```diff
@@ -162,10 +166,25 @@ def __xml__(self):
 
     @property
     def ephemeral(self):
-        """Should this volume be encrypted with an ephemeral key in dom0?
+        """Should this volume be encrypted with an ephemeral key in dom0
+        (if enabled with encrypted_volatile property)?
         """
-        return not self.snap_on_start and not self.save_on_stop and \
-                self.domain is None and self.rw
+        if self._ephemeral is not None:
+            return self._ephemeral
+        # default value
+        return False
+
+    @ephemeral.setter
+    def ephemeral(self, value):
+        if not value:
+            self._ephemeral = False
+            return
+        if self.snap_on_start or self.save_on_stop or \
+                self.domain is not None or not self.rw:
```

Comment:
Will that prevent DispVM templates from starting?

Reply:
It should not. It's about `volatile` volume only, which (currently) always have the same properties regardless of qube type. Furthermore, this is done on setting the property, not starting the qube.

Reply:
Ah, good catch :woman_facepalming: